Cargo.toml

```diff
@@ -0,0 +1,2 @@
+[workspace]
+members = ["native-packages/fs-search"]
```

Comment:
This could be `native-packages/*` if we're always putting Rust packages in here.